src/Main.cpp

```diff
@@ -195,7 +204,7 @@ int main(int argc, char *argv[])
 
     SDL_GL_SwapWindow(s_display);
     SDL_Delay(3000);
-    SDL_GL_DeleteContext(glContext);
```

Comment:
I don't see any evidence that SDL_DestroyWindow also deletes the GL context...  Am I missing something?

Reply:
Probably not.  I think this was an error I made conveniently masked by our currently-simplistic main function.